src/documents/SubcomponentDeclaration.md

```diff
@@ -163,9 +164,16 @@ to which it is injected has a path `child2.injectedComponent` in the final insta
 called an _injected subcomponent_ because it appears at a path other than the one where it was constructed. In memory, the
 JavaScript object references at path `concreteChild` and `child2.injectedComponent` will be identical.
 
+Injecting subcomponents can be useful in order to capture (or parameterise) variability in the relationship between two components, in
+order to expose it more cleanly to a third component. For example, a component `A` may be managed by a component `B` elsewhere in the tree,
```

Comment:
I'm not sure why `A` is managed by `B`, seeing as `B` is the subcomponent injected into `A`. I'm probably misconceptualizing something here so either further clarification, explanation, and/or a diagram could help.